File: C/7kyu/Growth of a Population/solution.c
int nbYear(int p0, double percent, int aug, int p) {
  int years = 0;
  percent = percent/100;
  while(p0 < p) {
    years++;
    p0 = p0 + (p0 * percent) + aug;
  }
  return years;
}

File: PHP/7kyu/Growth of a Population/solution.php
<?php
function nbYear($p0, $percent, $aug, $p) {
    $years = 0;
    $percent /= 100;
    
    while($p0 < $p) {
      ++$years;
      $p0 += $p0 * $percent + $aug;
    }
    
    return $years;
}
?>